requirements/edx/github.txt

```diff
@@ -68,6 +68,7 @@ git+https://github.com/hmarr/django-debug-toolbar-mongo.git@b0686a76f1ce3532088c
 git+https://github.com/edx/rfc6266.git@v0.0.5-edx#egg=rfc6266==0.0.5-edx
 # Used for testing
 git+https://github.com/edx/lettuce.git@0.2.20.002#egg=lettuce==0.2.20.002
+git+https://github.com/open-craft/django-storage-swift.git@eb812cb34a266a58558de4d05ff08b3afd025e36#egg=django-storage-swift==1.2.7
```

Comment:
@mollydb [django-storage-swift](https://github.com/blacktorn/django-storage-swift) uses the MIT licence (according to `setup.py`).

@edx/devops is it ok to include this here? I would prefer to include it separately for openstack deployments only, is there an easy way to do that?